app/views/_includes/claims/details.njk

```diff
@@ -41,7 +41,7 @@
             visuallyHiddenText: "accredited provider"
           }
         ]
-      } if claim.status == "draft"
+      } if claim.status == "draft" and claim.status === "submitted"
```

Comment:
Should this be `and` or `or`?
```suggestion
      } if claim.status in ["draft", "submitted"]
```

Reply:
This logic suggests that a claim can be edited even after submission. Is that correct?